13.0/s2i/bin/assemble

```diff
@@ -171,7 +172,43 @@ function configure_proxy() {
   xml="$xml\
        </proxy>"
     sed -i "s|<!-- ### configured http proxy ### -->|$xml|" $HOME/.m2/settings.xml
+    return
   fi
+
+  # if http(s)_proxy is set, use that.
+  proxy=${HTTPS_PROXY:-${https_proxy:=${HTTP_PROXY:=$http_proxy}}}
+  noproxy=${NO_PROXY:-$no_proxy}
+  if [ -n $proxy ]; then
+    proxyhost=$(echo $proxy | cut -d : -f 1,2)
+    proxyport=$(echo $proxy | cut -d : -f 3)
+    proxyprotocol="http"
+    if [[ `echo $proxy | grep -i https://` ]]; then
```

Comment:
It would give me the warm fuzzies if the // were escaped like \/\/

Reply:
it's grep, not sed, so there's no reason to...